app/models/menu.rb

```diff
@@ -1,5 +1,5 @@
 class Menu < ActiveRecord::Base
-  has_and_belongs_to_many :menu_categories
+  has_and_belongs_to_many :menu_items
```

Comment:
I would suggest `has_and_belongs_to_many :menu_items, -> { uniq }` as we only want a MenuItem to be added once to each Menu